minidump/src/registers.rs

```diff
@@ -1,8 +1,9 @@
 use symbolic_common::types::{Arch, CpuFamily, UnknownArchError};
+use gimli::Register;
 
 /// Returns the name of a register in a given architecture.
-pub fn get_register_name(arch: Arch, register: u8) -> Result<&'static str, UnknownArchError> {
-    let index = register as usize;
+pub fn get_register_name(arch: Arch, register: Register) -> Result<&'static str, UnknownArchError> {
```

Comment:
Can we still make this `Result<Option<...>>` and skip unknown registers in `write_register_rule`?